components/chat/chat-page.tsx

```diff
@@ -20,35 +21,32 @@ export default function ChatPage({ session, members, invitationsPage, dmChannelI
   const workspaceId = session.workspace
   return (
     <>
-      <div
-        className={`
-            fixed lg:relative inset-y-0 left-0 z-50 w-[300px] shadow-lg transform transition-transform duration-300 ease-in-out
-            
-          `}
-      >
-        <Sidebar
-          session={session}
-          selectedChannel={selectedChannel}
-          selectedDM={selectedDM}
-          setSelectedChannel={setSelectedChannel}
-          setSelectedDM={setSelectedDM}
-          members={members}
-          invitationsPage={invitationsPage}
-          dmChannelId={dmChannelId as string} 
-          user={user}
-        />
-      </div>
-
-      <div className="flex-1 flex flex-col">
-        <ChatView
-          selectedChannel={selectedChannel}
-          setSelectedChannel={setSelectedChannel}
-          selectedDM={selectedDM}
-          session={session}
-          workspaceId={workspaceId as string}
-          dmChannelId={dmChannelId as string} 
-        />
-      </div>
+      <ResizablePanelGroup direction="horizontal" className="h-screen w-full">
+        <ResizablePanel defaultSize={15} minSize={10} maxSize={50}>
```

Comment:
```suggestion
        <ResizablePanel defaultSize={15} minSize={15} maxSize={50}>
```

So we don't have this effect

<img width="572" height="370" alt="Image" src="https://github.com/user-attachments/assets/0106e94d-a5ad-4f9e-ae2f-4a2bdd3db920" />